assets/scss/style_variations.scss

```diff
@@ -30,6 +30,7 @@
 .nhsuk-care-card.nhsuk-care-card--type.is-style-warning-callout { /* Warning Callout */
```

Comment:
This seem a very specific selector?

Reply:
yes. Intentionally so. We dont want a yellow border on the other ones, just the warning callout